src/Map/index.tsx

```diff
@@ -156,6 +157,17 @@ const Map: React.FC<Props> = (props) => {
     geojsonSource?.setData(sourceJson as FeatureCollection)
   }, [sourceJson])
 
+  // Use to travel to location on card click
+  useEffect(() => {
```

Comment:
There's some interesting edge cases on this affect that I think will behave correctly but might need a comment. If `map == null` then we don't want to just to `zoomLocation` even once the map is loaded as it will jump to `initialLocation`? I think that's desired. Only when `zoomLocation` changes do we then want it to jump? Also the dependency list is extremely important here as adding more dependencies will make the xoom jump around when the use has just been scrolling. I can't think of a better way of doing this (so it's fine) but a comment on a coupld of these points may be helpful

Reply:
`map != null && props?.zoomLocation != null` This should hopefully stop any jumping around, unless the user clicks on the cards a lot. Even then I'm using `easeTo` so it won't jump it will transition over.